From 8de073c637a1ea968b7304f373a20b13a4a47bbc Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 12 Mar 2024 10:54:08 +0100
Subject: [PATCH] Create.c: include linux/falloc.h for FALLOC_FL_ZERO_RANGE
 definition

glibc provides this through fcntl.h but musl does not - should
be reported and fixed there.

Upstream-Status: Inappropriate [musl-specific issue]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 Create.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Create.c b/Create.c
index fd6c9215..3210a03e 100644
--- a/Create.c
+++ b/Create.c
@@ -33,6 +33,7 @@
 #include	<signal.h>
 #include	<sys/signalfd.h>
 #include	<sys/wait.h>
+#include	<linux/falloc.h>
 
 #ifndef FALLOC_FL_ZERO_RANGE
 #define FALLOC_FL_ZERO_RANGE 16
